### PR TITLE
Closes #32 add location, bio, and birthday to user table

### LINKED ISSUE #32: Add additional columns to Users table
Add bio, location and birthday (update migration file). Will also need to update seeds.

### PR DESCRIPTION
**Pull request by emilyaringoen**

### CODE REVIEW

**Comment by jenniferwiner** [comment]
@emilyaringoen Will take a look at this now.

File: server/migrations/20180920185810_users.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ exports.up = function(knex) {
     table.string('email').unique().notNullable();
     table.string('password').notNullable();
     table.string('avatar');
+    table.string('bio');
+    table.string('location');
+    table.date('birthday');
```
Comment (jenniferwiner) [inline server/migrations/20180920185810_users.js]: @emilyaringoen Thoughts on making any of these required? I think I like them optional bc then you don't have to require a user to submit all of their info when they create an account.

File: server/seeds/001_users.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (jenniferwiner) [inline server/seeds/001_users.js]: Killin it :) Removing that whitespace!

File: server/seeds/001_users.js [hunk 3]
```diff
@@ -9,13 +8,19 @@ exports.seed = function(knex) {
           name: 'Jane Doe',
           handle: 'janedoe',
           email: 'janedoe@gmail.com',
-          password: '$2b$10$M1hRH0FE1AJcqo4tT0uwkOWlnmxF6ewmWsmXW1lNhn/L7McVYnsHq' //password
+          password: '$2b$10$M1hRH0FE1AJcqo4tT0uwkOWlnmxF6ewmWsmXW1lNhn/L7McVYnsHq', //password,
+          bio: 'Artist. Juggler. React web developer.',
+          location: 'Boulder, CO',
+          birthday: '01/01/1990'
```
Comment (jenniferwiner) [inline server/seeds/001_users.js]: @emilyaringoen What date format does knex prefer?

File: server/seeds/001_users.js [hunk 4]
```diff
@@ -9,13 +8,19 @@ exports.seed = function(knex) {
           name: 'Jane Doe',
           handle: 'janedoe',
           email: 'janedoe@gmail.com',
-          password: '$2b$10$M1hRH0FE1AJcqo4tT0uwkOWlnmxF6ewmWsmXW1lNhn/L7McVYnsHq' //password
+          password: '$2b$10$M1hRH0FE1AJcqo4tT0uwkOWlnmxF6ewmWsmXW1lNhn/L7McVYnsHq', //password,
+          bio: 'Artist. Juggler. React web developer.',
```
Comment (jenniferwiner) [inline server/seeds/001_users.js]: I want this to be my bio

File: server/migrations/20180920185810_users.js (hunk 1 above)
Comment (emilyaringoen) [reply server/migrations/20180920185810_users.js]: These seemed to be good optional criteria -- not everyone is comfortable disclosing their birthday or location online.

File: server/seeds/001_users.js (hunk 3 above)
Comment (emilyaringoen) [reply server/seeds/001_users.js]: Knex (and sql in general??) accepts a wide variety on inputs for a date, but will translate them to YYYY-mm-dd format. This is what would be returned should we get that field.

**Comment by emilyaringoen** [comment]
@jenniferwiner all tests pass.

**Pull request merged**